src/uct/tcp/tcp_sockcm.c

```diff
@@ -22,9 +23,18 @@ ucs_config_field_t uct_tcp_sockcm_config_table[] = {
    "TCP CM private data length",
    ucs_offsetof(uct_tcp_sockcm_config_t, priv_data_len), UCS_CONFIG_TYPE_MEMUNITS},
 
+   UCT_TCP_SEND_RECV_BUF_FIELDS(auto, ucs_offsetof(uct_tcp_sockcm_config_t, sockopt)),
```

Comment:
maybe remove _default parameter, since in both case we pass "auto" just set it to "auto" inside the macro